test/staking/StakeManager.test.js

```diff
@@ -285,7 +285,7 @@ contract('StakeManager', async function (accounts) {
       stakerDetails.signer.toLowerCase().should.equal(user)
     })
 
-    it('should update and verify signer/pubkey', async function () {
+    it('should not update signer/pubkey twice', async function () {
```

Comment:
Please keep this test case, just override the limit value to 0. Add another test for the new code.

Reply:
done